components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -231,4 +342,48 @@ object WebExtensionSupport {
     }
 
     private fun WebExtension.toState() = WebExtensionState(id, url)
+
+    internal object ExtensionUpdater {
```

Comment:
Sine we only use this for `Addon`s, `feature-addons` is a better fit.

Reply:
👍